lib/mix/tasks/up_ex_doc_version.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Mix.Tasks.UpExDocVersion do
+  use Mix.Task
+
+  @shortdoc "Update mix.exs' ex_doc version"
+  @mix_exs "mix.exs"
+
+  def run([new_vsn]) do
+    content_up =
+      @mix_exs
+      |> File.read!()
+      |> run(new_vsn)
+
+    File.write!(@mix_exs, "#{content_up}\n")
+    env = [{"PATH", System.get_env("PATH")}]
+    System.cmd("mix", ["deps.unlock", "ex_doc"], env: env)
```

Comment:
We can do this and it should be fine, but probably more pragmatic is to run this, then execute `mix deps.get`  back in the shell (i.e., in whatever workflow that will use this). 

Thoughts?

Reply:
I don't mind doing it in another pull request @starbelly, even before we release. This fix is already scope creep caused by me, since it wasn't in the change proposed by @MarkoMin initially.

Reply:
@MarkoMin, if you prefer to do it, you can remove these calls to mix from here and set them in the CI bit... But lemme know, otherwise I can also do in a subsequent pr (since this one's working as-is)

Reply:
I can do it. Do I need to remove File:write also or just lines 14 and 15?

Reply:
done